modules/validation/binding.go

```diff
@@ -58,6 +60,33 @@ func addGitRefNameBindingRule() {
 	})
 }
 
+func addValidURLListBindingRule() {
+	// URL validation rule
+	binding.AddRule(&binding.Rule{
+		IsMatch: func(rule string) bool {
+			return strings.HasPrefix(rule, "ValidUrlList")
```

Comment:
Addressed in https://github.com/go-gitea/gitea/pull/32643/commits/e536a9df5f5f5023245be89fd3e08037f61e40a5

Reply:
But "why"?

Shouldn't we be strict for names?

Reply:
If we want a strict name, it should be `==` here. But I don't think `HasPrefix` is right here. For other examples, some maybe also not be right and some are for `xxx(` so they use `HasPrefix`.

Reply:
I agree, but why it was changed to `EqualFold`?

Reply:
I misunderstood how other rules implemented. I thought case insensitive seems reasonable. Now I think we should follow the old styles to keep consistency.

Reply:
I standardized on using `==` in https://github.com/go-gitea/gitea/pull/32643/commits/28615575ced8a2969abd3456d9a1dd5233d47e07. I agree that `HasPrefix` is not right, since it can cause `ValidUrl` to match `ValidUrlList`.